src/main/views/sso-user.njk

```diff
@@ -0,0 +1,60 @@
+{% from "macros/breadcrumb.njk" import breadcrumb %}
```

Comment:
Should rename this file to 'remove-user-sso.njk'

Reply:
fixed